MyClass2.cls

```diff
@@ -2,18 +2,17 @@ public class MyClass {
 
 	public static void foo1() {
 		for (Integer i=0; i<10; i++) {
+			
+			List<SObject> items = [Select Id, Name FROM account LIMIT 10];
```

Comment:
This data query is inside a loop. Please move the query outside the loop and put records....